OpenTabletDriver/InputDevice.cs

```diff
@@ -25,6 +29,10 @@ public InputDevice(IDriver driver, IDeviceEndpoint device, TabletConfiguration c
             Configuration = configuration;
             Identifier = identifier;
 
+            if (Configuration.Attributes.TryGetValue(DelayAttributeKeyName, out var delayStr))
+                if (!UInt32.TryParse(delayStr, out FeatureInitDelayMs))
+                    Log.Debug("Device", $"Could not parse '{delayStr}' from attribute {DelayAttributeKeyName}");
```

Comment:
```suggestion
                    Log.Write("Device", $"Could not parse '{delayStr}' from attribute {DelayAttributeKeyName}", LogLevel.Warning);
```

This should be Warning not Debug, as it shouldn't happen if the config is working.